config/packaging/mingw-w64-helics/PKGBUILD

```
@@ -3,27 +3,26 @@
 _realname=helics
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
-pkgver=2.0.0
-pkgrel=1
-pkgdesc="A C++ library for co-simulation"
+pkgver=2.3.1
+pkgrel=3
```

This should probably be 1, unless there have been previous releases of a 2.3.1 package.

```suggestion
pkgrel=1
```

---

probably right, I didn't really change that from the template I was using